lib/exception.py

```diff
@@ -70,6 +70,16 @@ class PackageDescriptorError(PackageError):
     error_code = 18
 
 
+class PackageDirectoryIsEmpty(PackageError):
+    DEFAULT_MESSAGE = "No packages to build. Packages directory is empty."
+    error_code = 19
+
+
+class PackageDirectoryIsEmpty(PackageError):
+    DEFAULT_MESSAGE = "No packages to build. Packages directory is empty."
+    error_code = 19
```

Comment:
This might have happened when I was fixing the merge conflicts. Thank you for noticing.